src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2930,6 +2931,10 @@ export class StudyViewPageStore {
         }
     });
 
+    readonly studyGoogleAnalytics = remoteData({
+        invoke: async () => await queryStudyAnalytics(this.studyIds, process.env.API_URL || ""),
```

Comment:
you actually can 1. take away async and 2. get rid of await since invoke just wants a promise returned.  we only really need async/await when we want to take advantage of the synrconous assignment sugar:
```var data = await getData();```

Reply:
Changes have since been made to implement this.